website/pages/en/new-chain-integration.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Integrating New Networks
+---
+
+Support for new on-chain data sources can be added to Graph Node, allowing developers to build and deploy subgraphs and substreams-based subgraphs for new chains. Two different interfaces can be used for blockchain data ingestion through Graph Node: EVM JSON-RPC and Firehose.
+
+**1. EVM JSON-RPC**
+
+If the blockchain is EVM equivalent and the client/node exposes the standard EVM JSON-RPC API, Graph Node should be able to index the new chain. For more information on testing, refer to [Integrating an EVM equivalent chain](new-chain-integration#integrating-an-evm-equivalent-chain).
+
+**2. Firehose**
+
+For non-EVM-based chains, Graph Node will need to ingest blockchain data via gRPC and known type definitions. This can be done via [Firehose](firehose/README/), a new technology developed by [StreamingFast](https://www.streamingfast.io/) that provides a highly-scalable indexing blockchain solution using a files-based and streaming-first approach.
+
+## Difference between EVM JSON-RPC & Firehose
+
+While the two are suitable for subgraphs, a Firehose is always required for developers wanting to build with [Substreams](substreams/), like building substreams-based subgraphs. In addition, Firehose allows for improved indexing speeds when compared to JSON-RPC.
+
+New EVM chain integrators may also consider the Firehose-based approach, given the benefits of substreams and its massive parallelized indexing capabilities. Supporting both allows developers to choose between building substreams or subgraphs for the new chain.
+
+> **NOTE**: A Firehose-based integration for EVM chains will still require Indexers to run the chain's archive RPC node to properly index subgraphs. This is due to the Firehose's inability to provide smart contract state typically accessible by the `eth_call` RPC method. (It's worth reminding that eth_calls are [not a good practice for developers](https://thegraph.com/blog/improve-subgraph-performance-reduce-eth-calls/))
+
+---
+
+## Integrating an EVM equivalent chain
```

Comment:
I feel like this isn't really integration so much as configuration

Reply:
Technically, yes. While we can move these configuration details elsewhere, I think it's important we reference it somehow so we have the important info for new chains (EVM) under this higher-level chain integration page. 
Wdyt?